indexer/database/bridge_transfers.go

```diff
@@ -141,7 +156,24 @@ l1_transaction_deposits.l2_transaction_hash`)
 		return nil, result.Error
 	}
 
-	return deposits, nil
+	hasNextPage := false
+	if len(deposits) > *limit {
+		hasNextPage = true
+		deposits = deposits[:*limit]
+	}
+
+	nextCursor := ""
```

Comment:
should this be a pointer to represent optional or does representing it with "" make sense?